packages/core/src/v3/runMetadata/metadataStream.ts

```
@@ -0,0 +1,88 @@
+        }
+        return queue.shift()!;
+      },
+    });
+
+    return {
+      serverIterator: teeIterator(this.serverQueue),
+      consumerIterator: teeIterator(this.consumerQueue),
+    };
+  }
+
+  private initializeServerStream(): Promise<void | Response> {
```

Biome

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)